backends/rapidpro/backend.go

```diff
@@ -410,28 +410,18 @@ func (b *backend) WriteExternalIDSeen(msg courier.Msg) {
 	writeExternalIDSeen(b, msg)
 }
 
-// ResolveMedia resolves the passed in attachment (content_type:url) to a media object
-func (b *backend) ResolveMedia(ctx context.Context, a string) (courier.Media, error) {
-	// split into content-type and URL
-	parts := strings.SplitN(a, ":", 2)
-	var contentType, mediaUrl string
-	if len(parts) <= 1 || strings.HasPrefix(parts[1], "//") {
-		return nil, errors.Errorf("invalid attachment format: %s", a)
-	}
-
-	contentType, mediaUrl = parts[0], parts[1]
-	stub := &DBMedia{URL_: mediaUrl, ContentType_: contentType}
-
-	// if we can't parse the URL or the hostname isn't our media domain, return stub
+// ResolveMedia resolves the passed in attachment URL to a media object
+func (b *backend) ResolveMedia(ctx context.Context, mediaUrl string) (courier.Media, error) {
```

Comment:
moved attachment string (content-type:url) parsing out of backend and into handler util